Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "stytch"
-version = "1.2.1"
+version = "1.3.0"
```

Comment:
tbh, there's enough in here to warrant a 2.0

1. PrimaryRequired moved from `b2b_discovery` to `b2b_sessions`
2. `email_id` type changed to `std::option::Option<string>` in many places
3. Overall, it's just a _big_ change with lots of new features

Let's not be afraid of bumping version numbers. I don't think anyone would be upset if we did a MAJOR update here and it also protects developers from cases where they think a MINOR wouldn't have any breaking changes.

Reply:
Sounds good, will change to major